packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1459,6 +1484,15 @@ class ListView extends BoxScrollView {
   /// the scroll position changes drastically.
   final double? itemExtent;
```

Comment:
Add a comment (maybe in a "See also:" section) about the prototypeItem property as an alternative.